registry.json

```
@@ -3,5 +3,10 @@
         "id": 1,
         "repository:": "https://github.com/paritytech/pallet-contracts-xcm",
         "typeName:": "Extension"
+    },
+    {
+        "id": 0x48f6,
```

JSON does not support hex.

```suggestion
        "id": 18678,
```